### ISSUE TITLE
Files in googleauth v0.6.0 have wrong permissions

### ISSUE THREAD

**Issue by ka2u** [google/google-auth-library-ruby#126]
Hi, I think .rb files should have `o+r` permission.
Please re-release fixed version.

```
[ka2u@348444a36e02] $ gem fetch googleauth -v 0.6.0
Fetching: googleauth-0.6.0.gem (100%)
Downloaded googleauth-0.6.0

[ka2u@348444a36e02] $ gem unpack googleauth-0.6.0.gem
Unpacked gem: '/home/ka2u/work/googleauth-0.6.0'

[ka2u@348444a36e02] $ cd googleauth-0.6.0
[ka2u@348444a36e02] $ ls
CHANGELOG.md  CONTRIBUTING.md  COPYING  Gemfile  README.md  Rakefile  googleauth.gemspec  lib  spec

[ka2u@348444a36e02] $ cd lib
[ka2u@348444a36e02] $ ls -al
drwxrwxr-x 3 ka2u ka2u 4096 10月 18 17:49 .
drwxrwxr-x 4 ka2u ka2u 4096 10月 18 17:49 ..
drwxrwxr-x 3 ka2u ka2u 4096 10月 18 17:49 googleauth
-rw-r----- 1 ka2u ka2u 1753 10月 18 17:49 googleauth.rb
```

**Comment by dazuma** [comment]
@ka2u Thanks for the report. We messed up the release, and will be pushing out a fixed 0.6.1 version momentarily.

**Comment by dazuma** [comment]
Fixed in googleauth 0.6.1
